components/atom/tag/src/index.js

```diff
@@ -80,7 +80,6 @@ AtomTag.propTypes = {
   /**
    * Custom classes
```

Comment:
I'd say this comment should be removed too, looks like its referring the className prop